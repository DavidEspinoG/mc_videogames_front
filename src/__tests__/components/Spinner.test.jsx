import Spinner from '../../components/spinner';
import renderWithProviders from '../test-utils';

describe('Spinner', () => {
  it('should render as expected', () => {
    const { container } = renderWithProviders(<Spinner />);
    expect(container).toMatchSnapshot();
  });
});
