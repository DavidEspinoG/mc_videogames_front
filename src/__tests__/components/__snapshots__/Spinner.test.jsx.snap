// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Spinner > should render as expected 1`] = `
<div>
  <svg
    height="300"
    preserveAspectRatio="xMidYMid"
    viewBox="0 0 100 100"
    width="300"
    xmlns="http://www.w3.org/2000/svg"
    xmlns:xlink="http://www.w3.org/1999/xlink"
  >
    <defs>
      <path
        d="M50 15A15 35 0 0 1 50 85A15 35 0 0 1 50 15"
        fill="none"
        id="electronPath"
      />
      <path
        d="M0 0A15 35 0 0 1 0 70A15 35 0 0 1 0 0"
        fill="none"
        id="electron"
      />
    </defs>
    <g>
      <animatetransform
        attributeName="transform"
        begin="0s"
        dur="5s"
        from="0 50 50"
        repeatCount="indefinite"
        to="360 50 50"
        type="rotate"
      />
      <mask
        id="layer-0-bottom-particle"
      >
        <rect
          fill="white"
          height="100"
          width="100"
          x="0"
          y="0"
        />
        <circle
          cx="52.5"
          cy="50"
          r="2.65"
        />
      </mask>
      <circle
        cx="52.5"
        cy="50"
        fill="#000"
        r="2.5"
        stroke="#fff"
        stroke-width="0.3"
      />
      <circle
        cx="48.75"
        cy="52.1650635094611"
        fill="#000"
        r="2.5"
        stroke="#fff"
        stroke-width="0.3"
      />
      <circle
        cx="48.75"
        cy="47.8349364905389"
        fill="#000"
        mask="url('#layer-0-bottom-particle')"
        r="2.5"
        stroke="#fff"
        stroke-width="0.3"
      />
    </g>
    <g>
      <animatetransform
        attributeName="transform"
        begin="0s"
        dur="5s"
        from="0 50 50"
        repeatCount="indefinite"
        to="360 50 50"
        type="rotate"
      />
      <mask
        id="layer-3.141592653589793-bottom-particle"
      >
        <rect
          fill="white"
          height="100"
          width="100"
          x="0"
          y="0"
        />
        <circle
          cx="47.5"
          cy="50"
          r="2.65"
        />
      </mask>
      <circle
        cx="47.5"
        cy="50"
        fill="#000"
        r="2.5"
        stroke="#fff"
        stroke-width="0.3"
      />
      <circle
        cx="51.25"
        cy="47.8349364905389"
        fill="#000"
        r="2.5"
        stroke="#fff"
        stroke-width="0.3"
      />
      <circle
        cx="51.25"
        cy="52.1650635094611"
        fill="#000"
        mask="url('#layer-3.141592653589793-bottom-particle')"
        r="2.5"
        stroke="#fff"
        stroke-width="0.3"
      />
    </g>
    <g
      data-testid="electron"
      transform="rotate(0 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#97bf0f"
        r="2"
      >
        <animatemotion
          begin="-1.4909054575479577s"
          dur="1.1777495908121638s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(0 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#10bbb5"
        r="2"
      >
        <animatemotion
          begin="-1.3731304984667414s"
          dur="1.1777495908121638s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(0 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#000"
        r="2"
      >
        <animatemotion
          begin="-1.255355539385525s"
          dur="1.1777495908121638s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(0 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#97bf0f"
        r="2"
      >
        <animatemotion
          begin="-1.1375805803043084s"
          dur="1.1777495908121638s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(0 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#10bbb5"
        r="2"
      >
        <animatemotion
          begin="-1.0198056212230922s"
          dur="1.1777495908121638s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(0 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#000"
        r="2"
      >
        <animatemotion
          begin="-0.9020306621418758s"
          dur="1.1777495908121638s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(0 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#97bf0f"
        r="2"
      >
        <animatemotion
          begin="-0.7842557030606594s"
          dur="1.1777495908121638s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(0 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#10bbb5"
        r="2"
      >
        <animatemotion
          begin="-0.666480743979443s"
          dur="1.1777495908121638s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(0 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#000"
        r="2"
      >
        <animatemotion
          begin="-0.5487057848982266s"
          dur="1.1777495908121638s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(0 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#97bf0f"
        r="2"
      >
        <animatemotion
          begin="-0.43093082581701014s"
          dur="1.1777495908121638s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(60 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#10bbb5"
        r="2"
      >
        <animatemotion
          begin="-1.9008448774711033s"
          dur="1.180445379505871s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(60 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#000"
        r="2"
      >
        <animatemotion
          begin="-1.7828003395205163s"
          dur="1.180445379505871s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(60 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#97bf0f"
        r="2"
      >
        <animatemotion
          begin="-1.664755801569929s"
          dur="1.180445379505871s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(60 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#10bbb5"
        r="2"
      >
        <animatemotion
          begin="-1.546711263619342s"
          dur="1.180445379505871s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(60 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#000"
        r="2"
      >
        <animatemotion
          begin="-1.428666725668755s"
          dur="1.180445379505871s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(60 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#97bf0f"
        r="2"
      >
        <animatemotion
          begin="-1.3106221877181679s"
          dur="1.180445379505871s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(60 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#10bbb5"
        r="2"
      >
        <animatemotion
          begin="-1.1925776497675808s"
          dur="1.180445379505871s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(60 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#000"
        r="2"
      >
        <animatemotion
          begin="-1.0745331118169936s"
          dur="1.180445379505871s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(60 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#97bf0f"
        r="2"
      >
        <animatemotion
          begin="-0.9564885738664066s"
          dur="1.180445379505871s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(60 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#10bbb5"
        r="2"
      >
        <animatemotion
          begin="-0.8384440359158196s"
          dur="1.180445379505871s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(120 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#000"
        r="2"
      >
        <animatemotion
          begin="-2.094196697238161s"
          dur="1.0547946515841289s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(120 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#97bf0f"
        r="2"
      >
        <animatemotion
          begin="-1.9887172320797482s"
          dur="1.0547946515841289s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(120 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#10bbb5"
        r="2"
      >
        <animatemotion
          begin="-1.8832377669213354s"
          dur="1.0547946515841289s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(120 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#000"
        r="2"
      >
        <animatemotion
          begin="-1.7777583017629226s"
          dur="1.0547946515841289s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(120 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#97bf0f"
        r="2"
      >
        <animatemotion
          begin="-1.6722788366045096s"
          dur="1.0547946515841289s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(120 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#10bbb5"
        r="2"
      >
        <animatemotion
          begin="-1.5667993714460966s"
          dur="1.0547946515841289s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(120 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#000"
        r="2"
      >
        <animatemotion
          begin="-1.4613199062876838s"
          dur="1.0547946515841289s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(120 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#97bf0f"
        r="2"
      >
        <animatemotion
          begin="-1.355840441129271s"
          dur="1.0547946515841289s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(120 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#10bbb5"
        r="2"
      >
        <animatemotion
          begin="-1.250360975970858s"
          dur="1.0547946515841289s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
    <g
      data-testid="electron"
      transform="rotate(120 50 50)"
    >
      <circle
        cx="50"
        cy="15"
        fill="#000"
        r="2"
      >
        <animatemotion
          begin="-1.144881510812445s"
          dur="1.0547946515841289s"
          repeatCount="indefinite"
        >
          <mpath
            href="#electron"
          />
        </animatemotion>
      </circle>
    </g>
  </svg>
</div>
`;
